app/index.py

```diff
@@ -11,19 +10,50 @@
 app.secret_key = get_config().app_secret_key
 
 
+def authenticated(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if 'authenticated' not in session:
+            return redirect(url_for('login'))
+        return func(*args, **kwargs)
+
+    return wrapper
```

Comment:
to powinno się znaleźć w domenie common w poddomenie "auth"

Reply:
Teraz jest w `app.domain.common.authentication.decorators.logged_in.py`